tox.ini

```diff
@@ -1,5 +1,16 @@
 [tox]
-envlist = lint,py{27,34,35,36,37,38,39,34-no-typing},pypy,pypy3,manifest,coverage-report
+envlist = lint,py{27,35,36,37,38,39},pypy,pypy3,manifest,coverage-report
+
+[gh-actions]
+python =
+    2.7: py27, coverage-report
+    3.5: py35, coverage-report
+    3.6: py36, coverage-report
+    3.7: py37, coverage-report
+    3.8: py38, coverage-report
+    3.9: py39, mypy, lint, manifest, coverage-report
+    pypy2: pypy, coverage-report
+    pypy3: pypy3, coverage-report
```

Comment:
Do we really need the `coverage-report` environment anymore?  My impression is that it creates duplicate Coveralls reports in PRs.  I think this is not necessary since the GitHub action you added is supposed to communicate with coveralls and the only thing we need to do is to run the tests with coverage enabled (which is already the case for the `py27`, `py35`, ..., environments.